tuna/example/example_lib.py

```diff
@@ -45,15 +47,16 @@ def __init__(self):
     super().__init__(library=Library.EXAMPLE)
     self.args = None
 
-  def parse_args(self):
+  def parse_args(self) -> None:
     # pylint: disable=too-many-statements
     """Function to parse arguments"""
+    parser: argparse.Namespace
     parser = setup_arg_parser('Example library integrated with MITuna', [
         TunaArgs.ARCH, TunaArgs.NUM_CU, TunaArgs.VERSION, TunaArgs.SESSION_ID,
         TunaArgs.MACHINES, TunaArgs.REMOTE_MACHINE, TunaArgs.LABEL,
         TunaArgs.RESTART_MACHINE, TunaArgs.DOCKER_NAME
     ])
-    group = parser.add_mutually_exclusive_group()
+    group: argparse.Namespace = parser.add_mutually_exclusive_group()
```

Comment:
perhaps `group: argparse._MutuallyExclusiveGroup`? But it appears all the parents of group are private classes:
``` 
>>> inspect.getmro(group.__class__) 
(<class 'argparse._MutuallyExclusiveGroup'>, <class 'argparse._ArgumentGroup'>, <class 'argparse._ActionsContainer'>, <class 'object'>)
```